tests/ci/yumrepo.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+import sys
+
+import requests
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+def main() -> None:
+    repodir = "/repo"
+    os.mkdir(f"{repodir}")
+    with open("./package_list/c8s-image-manifest.txt") as im:
+        for p in im.readlines():
+            pkg = f"{p.strip()}.el8.aarch64.rpm"
```

Comment:
"el8" should be part of the nvr read from "c8s-image-manifest.txt" . It misses those. I will need to regenerate that list again without stripping "el8" so that you don't have to add it here

Reply:
Removed _el8_